library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -113,39 +110,18 @@ def main
       # Predefined network device regular expressions
       @DeviceRegex = {
         # device types
-        "netcard" => Ops.add(
-          Ops.add(
-            Ops.get(@CardRegex, "netcard", ""),
-            HotplugRegex(["ath", "eth", "tr", "wlan"])
-          ),
-          "|usb-usb|usb-usb-"
-        ),
-        "modem"   => Ops.get(@CardRegex, "modem", ""),
-        "isdn"    => Ops.add(
-          Ops.get(@CardRegex, "isdn", ""),
-          HotplugRegex(["isdn", "ippp"])
-        ),
-        "dsl"     => Ops.get(@CardRegex, "dsl", ""),
+        "netcard" => @CardRegex["netcard"],
+        "modem"   => @CardRegex["modem"],
+        "isdn"    => @CardRegex["isdn"],
+        "dsl"     => @CardRegex["dsl"],
```

Comment:
modem, isdn and dsl support was removed a long time ago. I think this can be dropped.

Reply:
OK. I was not sure how far I should go in this bugfix. Should we do it here?

Reply:
well you touch it anyway, so why not.